mocked-api/handlers.ts

```diff
@@ -180,36 +210,278 @@ export const handlers = [
 
     return res(ctx.status(202), ctx.delay(apiDelay), ctx.json({}));
   }),
+
+  // processors
+  // get all processors of a bridge
+  rest.get(`${apiUrl}/bridges/:bridgeId/processors`, (req, res, ctx) => {
```

Comment:
Here be dragons.. 

The real backend will return `ItemNotFoundException` (i.e. code 4) if the `Bridge` does not exist.

Reply:
Done with [cf0c44c](https://github.com/5733d9e2be6485d52ffa08870cabdee0/sandbox-ui/pull/30/commits/cf0c44cf68e6801947e0109eba0690a8ccd93925)